services/QuillLMS/db/migrate/20240328190319_add_coda_to_evidence_research_gen_ai_llm_prompt_templates.evidence.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# This migration comes from evidence (originally 20240328185342)
+class AddCodaToEvidenceResearchGenAILLMPromptTemplates < ActiveRecord::Migration[7.0]
+  def change
+    add_column :evidence_research_gen_ai_llm_prompt_templates,
+      :coda,
```

Comment:
I'm assuming you are using `coda` since it's added at the end (or is this an ML term I'm not aware of). We might consider using plainer language. It's not a particularly common word (I only know it from crossword puzzles).

Reply:
It's not an ML word, I borrowed it from linguistics, (I thought of it by pronouncing CoT), but in retrospect, I agree with your point.